documentation/index_JuliasVersion.md

```diff
@@ -6,36 +6,44 @@ title: SpecmateDoc
+* Stellen Sie sicher, dass Java 11 installiert ist. Wenn nicht, besorgen Sie es sich z.B. von [hier](https://adoptopenjdk.net/). Um herauszufinden, welche Java-Version Sie gerade verwenden, geben Sie `java -version` in Ihre Konsole ein.
 * Besorgen Sie sich die neueste Version von Specmate auf der Download-Seite. Wo und wie Sie die Datei herunterladen, lesen Sie im nächsten Abschnitt *Konfiguration*.
-* Führen Sie ```java -jar prod-specmate-all.jar --configurationFile /pfad/zur/config/datei``` in Ihrer Konsole aus.
 * Wir empfehlen Ihnen, für Specmate den Browser "Google Chrome" zu verwenden, da Specmate mit Chrome am besten funktioniert und die Wahrscheinlichkeit von Darstellungsfehlern so minimiert wird. [Hier](https://www.google.com/intl/de_de/chrome/) können Sie Chrome downloaden. Sollten Sie eine zu alte Browserversion verwenden, zeigt Ihnen Specmate eine Warnung an.
 
 ## Konfiguration
-Specmate wird über die Datei *specmate-config.properties* konfiguriert. Eine Beispiel-Konfigurationsdatei erhalten Sie [hier](https://github.com/junkerm/specmate/blob/develop/bundles/specmate-config/config/specmate-config.properties).
-Kopieren Sie den angezeigten Code in einen Texteditor und speichern Sie die Datei mit der Endung *.properties*. Somit haben Sie Zugriff auf die aktuellste Version von Specmate.
+- Laden Sie sich auf der [Specmate-Homepage](https://specmate.io/?page_id=14) die neueste Version von Specmate herunter.
+- Entpacken Sie die Zip-Datei und starten Sie die in dem Ordner erhaltene Stapelverarbeitungsdatei mithilfe Ihrer Konsole. Sie erkennen diese Datei an der Endung ".sh" oder ".bat" abhängig von Ihrem Betriebssystem (sh bei Mac und Linux und bat bei Windows).
 
 ## Start von Specmate
-Specmate ist ein web-basiertes Werkzeug. Um Specmate zu starten, öffnen Sie Ihre Konsole und geben Sie Folgendes ein:
+Specmate ist ein web-basiertes Werkzeug. Um Specmate zu starten, öffnen Sie Ihren Browser (am besten Chrome) und navigieren Sie zu http://localhost:8080, um auf die Startseite von Specmate zuzugreifen.
 
-``properties java -jar specmate.jar --configurationFile [Pfad-zu-ihrer-config-file]```
+## Login
+Nach dem Aufruf von Specmate wird Ihnen die Anmeldeseite angezeigt. Bitte geben Sie hier einen Benutzernamen, ein Passwort und ein Specmate-Projekt ein. Für die Verwendung von Specmate benötigen Sie keinen dezidierten Login, stattdessen können Sie die Anmeldedaten der Anforderungsquelle, die mit dem Specmate-Projekt verbunden ist, verwenden.
 
-Jetzt können Sie einen Browser öffnen und zu http://localhost:8080 navigieren, um auf die Startseite von Specmate zuzugreifen.
+## Login via JIRA
 
-## Login
+Wenn Sie Specmate für Ihr Jira-Projekt verwenden wollen, loggen Sie sich bei Jira ein. Unten rechts auf Ihrer Projekt-Seite sehen Sie Specmate und ein Dropdown-Menü. Wenn Sie auf dieses klicken, erscheint ein Link zur Anmeldung bei Specmate via Jira.
```

Comment:
Das konkrete Projekt in Specmate kann auch anders heißen, je nachdem welcher Name in Specmate konfiguriert ist. Das bei uns das Specmate-Projekt auch "jira" heißt ist nur Zufall. Man muss also quasi wissen, dass das Specmate Projekt xyz ein Jira Projekt ist und man sich mit seinen Jira zugangsdaten anmelden kann. Guter Punkt- wenn ich drüber nachdenke wäre es nicht schlecht, das irgendwie in der UI anzuzeigen, z.B. mit einem Jira Logo...